### PR TITLE
Sensor reading closes #4

### LINKED ISSUE #4: Sensor reading
As a self-driving car, I would like to report my IR & ultrasonic sensors to my kind programmer overlords so that I can get controlled by them.

Acceptance criteria: Successfully reported IR & ultrasonic sensor readings ~~to MQTT~~.

### PR DESCRIPTION
**Pull request by ktpa**

### CODE REVIEW

File: arduino/smartcar/smartcar.ino [hunk 1]
```diff
@@ -1,9 +1,66 @@
...
+
+
 
-void loop() {
-  // put your main code here, to run repeatedly:
+SmartCar car(arduinoRuntime, control, gyroscope, leftOdometer, rightOdometer);
+
+const auto triggerPin = 6;
+const auto echoPin = 7;
+const auto maxDistance = 400;
```
Comment (dekmm) [inline arduino/smartcar/smartcar.ino]: We should keep naming consistent. If you've used uppercase for constants already, then these should follow that naming convention as well.

File: arduino/smartcar/smartcar.ino [hunk 2]
```diff
@@ -1,9 +1,66 @@
...
+unsigned long previousPrintout     = 0;
+
+ArduinoRuntime arduinoRuntime;
+BrushedMotor leftMotor(arduinoRuntime, smartcarlib::pins::v2::leftMotorPins);
+BrushedMotor rightMotor(arduinoRuntime, smartcarlib::pins::v2::rightMotorPins);
+DifferentialControl control(leftMotor, rightMotor);
+
+GY50 gyroscope(arduinoRuntime, 37);
+
+const auto pulsesPerMeter = 600;
```
Comment (dekmm) [inline arduino/smartcar/smartcar.ino]: Same naming suggestion as bellow

**Comment by platisd** [comment]
Just a note here, the `closes #4` statement should be in the description of the pull request, not the title.

**Comment by ktpa** [comment]
Thank you Platisd, it's the first time I'm using it. Usually linking PR to issues is no problem but I guess the way we setup this project doesn't allow for that.

**Comment by platisd** [comment]
> but I guess the way we setup this project doesn't allow for that.

Not sure I follow what that means :P

**Comment by ktpa** [comment]
image
I couldn't actually connect by UI, so that's why I tried. It's connected now though.

**Review by gsastraw — APPROVED** [review]

**Review by JensTheS — APPROVED** [review]
Works like a charm.. 😄 
image

**Pull request merged**